src/extent.c

```diff
@@ -216,7 +216,10 @@ extents_insert_locked(tsdn_t *tsdn, extents_t *extents, extent_t *extent,
 		extent_list_append(&extents->lru, extent);
 	}
 	size_t npages = size >> LG_PAGE;
-	atomic_add_zu(&extents->npages, npages);
+	size_t cur_extents_npages =
+	    atomic_load_zu(&extents->npages, ATOMIC_RELAXED);
+	atomic_store_zu(&extents->npages, cur_extents_npages + npages,
```

Comment:
How is splitting an atomic add into an atomic load followed by an atomic store not racy?  Maybe if the load were ```ATOMIC_ACQUIRE``` and the store were ```ATOMIC_RELEASE``` this would be okay?